src/corehost/cli/fxr/sdk_resolver.cpp

```diff
@@ -64,7 +64,7 @@ pal::string_t resolve_cli_version(const pal::string_t& global_json)
     return retval;
 }
 
-pal::string_t resolve_sdk_version(pal::string_t sdk_path, bool parse_only_production, pal::string_t global_cli_version)
+pal::string_t resolve_sdk_version(pal::string_t sdk_path, bool disallow_prerelease, pal::string_t global_cli_version)
```

Comment:
Pass `disallow_prerelease` by reference? The calling functions pass by value, but don't have the check for `if (specified.is_prerelease())` like below

Reply:
I'd like to leave this as is.

I consider the fact that disallow_prerelease is overridden when global.json has a preview version specified to be an implementation detail of this function. As of now, the caller has no need to observe this. I also find in/out parameters confusing.

Reply:
So here was a bug here, but I fixed it by removing disallow_prerelease as a parameter where it does not make sense to ever disallow prerelease. Still prefer to avoid in/out param for this. 

See 03ab2be